odata-url-conventions/5 Query Options.md

```diff
@@ -2061,6 +2061,8 @@ Allowed system query options are
 [`$count`](#SystemQueryOptioncount), and
 [`$search`](#SystemQueryOptionsearch)
 for collection-valued navigation properties.
+The same system query option, irrespective of casing or whether or not it is prefixed with a `$`,
+MUST NOT be specified more than once for a collection-valued navigation property.
```

Comment:
Why only collection-valued? Are duplicate `$compute` and `$expand` allowed after single-valued navigation properties?